tests/test_plugins/test_crypto_plugins/test_ecb.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from qark.plugins.crypto.ecb_cipher_usage import ECBCipherCheck
+from qark.plugins.crypto.packaged_private_keys import PackagedPrivateKeys
+from qark.plugins.crypto.setting_secure_random_seed import SeedWithSecureRandom
+
+curr_dir = os.path.dirname(__file__)
+java_dir = os.path.join(curr_dir, 'java_files')
+
+
+
+def test_non_existent_file():
+    """The scan should not throw exceptions on a non-existent file"""
+    plugin = ECBCipherCheck()
+    non_existent_files = ['----']
+    try:
+        plugin.run(non_existent_files)
+    except:
+        assert False
```

Comment:
Why is this here? if we want the test to fail on a plugin fail we could just remove `plugin.run` and let it throw an Exception in the test.

Reply:
This is to make sure that the plugin doesn't bubble up the exception